### DISCUSSION TITLE
Bug - Validations don't run on draft orders.

### DISCUSSION THREAD

**Discussion by ct-Marvin** [Shopify/function-examples#473] [Feedback: Cart and Checkout Validation API]
There are a lot of apps using draft orders for example:
https://apps.shopify.com/ymq-options

All checkouts created including one of those draft order products do never trigger a validation. 

This lead to multiple merchants already uninstalling our validation app.

**Comment by nickwesselman** [comment]
This is a [documented limitation](https://shopify.dev/docs/api/functions/reference/cart-checkout-validation). There's an [existing feature request here](https://github.com/Shopify/function-examples/discussions/393) which I'd recommend upvoting and commenting on, with a description of your use case.
